recipe/test/myrun_test.py

```diff
@@ -12,9 +12,10 @@
 
 except ImportError:
     print ('JCC Import Error')
```

Comment:
i'd still recommend printing out the error in case bad things happen

Reply:
yep! thanks!